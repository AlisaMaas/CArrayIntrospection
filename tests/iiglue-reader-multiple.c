#include "iiglue-reader-peek.h"
#include "iiglue-reader-variadic.h"

int multiple(int *values)
{
	variadic(values);
	return peek(values, 0);
}
