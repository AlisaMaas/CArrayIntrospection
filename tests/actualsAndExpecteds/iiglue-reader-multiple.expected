Printing analysis 'Read iiglue analysis results and add them as metadata on corresponding LLVM entities':
	array arguments:
		peek::0
		variadic::0
